cmd/internal/kube/secret.go

```diff
@@ -0,0 +1,127 @@
+package kube
+
+import (
+	"encoding/base64"
+	"log"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/akitasoftware/akita-cli/printer"
+
+	"github.com/akitasoftware/akita-cli/cmd/internal/cmderr"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var (
+	output    string
+	namespace string
```

Comment:
Our convention for flag variables seems to be outputFlag, namespaceFlag

Reply:
I've updated these flag names in: [69857a4](https://github.com/akitasoftware/akita-cli/pull/202/commits/69857a46fa69b814319c8ebe9ca018b97445e366)